copilot/data-store-celery/manifest.yml

```diff
@@ -0,0 +1,55 @@
+# The manifest for the "data-store-celery" service.
+# Read the full specification for the "Backend Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/backend-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: data-store-celery
+type: Backend Service
+
+# Your service does not allow any traffic.
+
+# Configuration for your containers and service.
+image:
+
+# TODO: re-enable this when secrets are tied up properly Celr
+#  healthcheck:
+#    command: ['CMD-SHELL', 'launcher', 'celery', '-A', 'app.celery_app', 'inspect', 'ping']
```

Comment:
Can we restore this healthcheck?

Reply:
I'm working on it but the values need tweaking a bit.